cms/models_test.py

```diff
@@ -86,6 +90,41 @@ def test_notification_snippet():
     assert str(notification) == message_text
 
 
+def test_webinar_course():
+    """
+    Verify `course` property from the webinar page returns expected value
+    """
+    course = CourseFactory.create()
+    webinar_page = WebinarPageFactory.create(course=course)
+    assert webinar_page.course == course
+
+
+def test_upcoming_webinar_date_time():
+    """
+    Verify `date` and `time` property for the upcoming webinar to be present
+    """
+    todays_date = datetime.today()
+    future_date = todays_date + timedelta(days=10)
+    webinar = WebinarPageFactory.create(date=future_date, time="11 am")
+
+    assert webinar.category == UPCOMING_WEBINAR
+    assert webinar.date == future_date.date()
+    assert webinar.time == "11 am"

```

Comment:
Can you make this a parameterized test? e.g. you can send different value combinations along with `None`. And make sure the behavior is intact. Right now it only checks for a happy scenario.

Could you also check for other places like this and make other tests parametrized as well?